File: mentor-on-demand-system-mentor-skill-ms/src/main/java/com/ibm/mods/mentorskill/model/MentorSkill.java
package com.ibm.mods.mentorskill.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "mentor_skills")
public class MentorSkill {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private long id;
	
	@Column(name = "mentor_id")
	private long mentorId;
	
	@Column(name = "skill_id")
	private long skillId;
	
	@Column(name = "self_rating")
	private int selfRating;
	
	@Column(name = "years_of_exp")
	private int yearOfExperience;
	
	@Column(name = "facilities_offered")
	private String facilitiesOffered;

	@Column(name = "hourly_rate")
	private double hourlyRate;
	
	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public long getMentorId() {
		return mentorId;
	}

	public void setMentorId(long mentorId) {
		this.mentorId = mentorId;
	}

	public long getSkillId() {
		return skillId;
	}

	public void setSkillId(long skillId) {
		this.skillId = skillId;
	}

	public int getSelfRating() {
		return selfRating;
	}

	public void setSelfRating(int selfRating) {
		this.selfRating = selfRating;
	}

	public int getYearOfExperience() {
		return yearOfExperience;
	}

	public void setYearOfExperience(int yearOfExperience) {
		this.yearOfExperience = yearOfExperience;
	}

	public String getFacilitiesOffered() {
		return facilitiesOffered;
	}

	public void setFacilitiesOffered(String facilitiesOffered) {
		this.facilitiesOffered = facilitiesOffered;
	}

	public double getHourlyRate() {
		return hourlyRate;
	}

	public void setHourlyRate(double hourlyRate) {
		this.hourlyRate = hourlyRate;
	}

}

File: mentor-on-demand-app-client/src/app/models/mentor-skill.ts
export class MentorSkill {
    
    id: number;
    mentorId: number;
    skillId: number;
    selfRating: number;
    yearOfExperience: number;
    facilitiesOffered: string;
    hourlyRate: number;

    constructor(id: number
            ,   mentorId: number
            ,   skillId: number
            ,   selfRating: number
            ,   yearOfExperience: number
            ,   facilitiesOffered: string
            ,   hourlyRate: number) {

                this.id = id;
                this.mentorId = mentorId;
                this.skillId = skillId;
                this.selfRating = selfRating;
                this.yearOfExperience = yearOfExperience;
                this.facilitiesOffered = facilitiesOffered;
                this.hourlyRate = hourlyRate;
        }

    get Id(): number {
        return this.id;
    }

    set Id(id: number) {
        this.id = id;
    }

    get MentorId(): number {
        return this.mentorId;
    }

    set MentorId(mentorId: number) {
        this.mentorId = mentorId;
    }

    get SkillId(): number {
        return this.skillId;
    }

    set SkillId(skillId: number) {
        this.skillId = skillId;
    }

    get SelfRating(): number {
        return this.selfRating;
    }

    set SelfRating(selfRating: number) {
        this.selfRating = selfRating;
    }

    get YearOfExperience(): number {
        return this.yearOfExperience;
    }

    set YearOfExperience(yearOfExperience: number) {
        this.yearOfExperience = yearOfExperience;
    }

    get FacilitiesOffered(): string {
        return this.facilitiesOffered;
    }

    set FacilitiesOffered(facilitiesOffered: string) {
        this.facilitiesOffered = facilitiesOffered;
    }

    get HourlyRate(): number {
        return this.hourlyRate;
    }

    set HourlyRate(hourlyRate: number) {
        this.hourlyRate = hourlyRate;
    }

}
